client/src/app/pages/applications/application-assessment/components/application-assessment-wizard/application-assessment-wizard.tsx

```diff
@@ -229,8 +236,8 @@ export const ApplicationAssessmentWizard: React.FC<
             break;
           case SAVE_ACTION_VALUE.SAVE_AND_REVIEW:
             assessment &&
-              getApplicationById(assessment?.applicationId)
-                .then(({ data }) => {
+              getApplicationById(assessment?.application?.id || 0)
```

Comment:
Is `0` a good default application id to use here if the assessment doesn't have an application (?) or if the application doesn't have an id (?)?